Update functionality to use latest SlimPHP and Vonage PHP SDK

GregHolmes opened the pull request:
In order to update the existing blog post, I've updated the code snippet found here. 
This has involved updating the version of SlimPHP. As well as replacing the Nexmo PHP SDK with the Vonage PHP SDK. Which also has changes to its functionality, such as NCCOs are no longer arrays but classes.

lornajane reviewed (changes requested):
Nice to see some of the existing stuff getting an update!! Not sure we are quite there yet (the repo itself needs a description and topics added as well)

This README would benefit from some more updates
 - the `ngrok` command is incorrect 
 - tell the user what to update their URLs to (looks like `/webhooks/answer` and maybe we should add an event one that logs what it receives)
 - "request" is misspelled
 - Does the user need to create an application? Get some creds? Something else? Do those need to go somewhere (looks like hardcoded into the file but let's make that clear!)
 - what is an `<ID>` and where can the user get that number?  If the ID should be the call ID then let's say so and include instructions on finding that in the ngrok interface.

This application also (minor point!) does not work for me. Assuming I'm supposed to be constructing a URL like `http://localhost:8000/trigger/dcee6420d4f66defdf25305414c6c5c1/1` it all seems very happy about itself but I don't hear anything in the call.

GregHolmes commented:
Thank you for your review @lornajane  I've updated the README to fix all of the issues you've raised. I've also added a handler for the /webhooks/event webhook in the code to output the status as and when it happens rather than the logs have errors.

Please let me know what you think of the revised README doc.

lornajane commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,132 @@
+# Contributing Guidelines
```
Where did this come from? I'm not sure I dislike it but it's pretty long and I'm not sure all of it applies to this project, might be worth an edit, or checking the Nexmo/Vonage standard examples https://github.com/Nexmo/repo-standards

GregHolmes replied:
I may have borrowed it from repo we have. But I'm afraid I can't remember which one!

lornajane reviewed (approved):
Love it! Minor query about one of the supporting files but I'll approve anyway.

GregHolmes commented:
Thanks Lorna. I'm going to merge this as the repo-standards for a supporting repo for a blog post, it advises a lot less than what we have.
